command/stats_test.go

```diff
@@ -0,0 +1,89 @@
+package command_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/ghttp"
+	. "github.com/openebs/maya/command"
+	"os"
+)
+
+var _ = Describe("GetVolAnnotations", func() {
```

Comment:
Anything that is not specific to go std lib can be explained in forms of code comments.